net.labymod.LabyMod.yml

```diff
@@ -0,0 +1,36 @@
+id: net.labymod.LabyMod
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: labymodlauncher
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --env=ELECTRON_DISABLE_SANDBOX='1'
+  - --persist=.minecraft
+modules:
+  - name: LabyMod
+    buildsystem: simple
+    build-commands:
+       - bsdtar -xf laby.deb
+       - tar -xf data.tar.xz
+       - install -D --mode 0644 usr/lib/labymodlauncher/resources/icons/icon.png /app/share/icons/hicolor/scalable/apps/net.labymod.LabyMod.png
```

Comment:
a PNG is NOT scalable.

Reply:
no. this is not Windows.

Scalable icons are in SVG. Not scalable icons are in the directory matching their size. Note that size are standardized, not arbritrary.

Reply:
Does it fit now?